configure.ac

```diff
@@ -1347,6 +1347,80 @@ case "$target_os" in
         ;;
 esac
 
+dnl ######################################################################
+dnl Historically process-handling has been based on invoking the 'ps' command
+dnl and parsing its output.  This has always been fiddly and error-prone.
+dnl
+dnl Most UNIX/Linux systems offer more machine-oriented mechanisms, for instance:
+dnl   o  Linux: /proc
+dnl   o  Solaris: /proc (different details)
+dnl   o  FreeBSD: kvm_getprocs()
+dnl
+dnl We have begun (2019) to investigate using these on a per-platform basis.
+dnl Because there is variety, we refer to this by the general term "system probe".
+dnl The previous ps-cmd mechanism is retained as a fallback for platforms.
+dnl ######################################################################
+
+dnl These are the supported platforms:
+dnl   o  Legacy ps-output support as before;
+dnl   o  Newer "system probe" ("sysprobe") not yet implemented for all.
+
+dnl For BSD ensure we have "kvm_getprocs()"
+case "$target_os" in
+  *bsd*|dragonfly*)
+    AC_CHECK_LIB([kvm], [kvm_getprocs],
+        [AC_DEFINE([HAVE_LIBKVM], 1, [Define if -lkvm is available])
+        LIBS="-lkvm $LIBS"],
+        AC_MSG_ERROR(Cannot find kvm library required for BSD)
+    )
+    ;;
+esac
```

Comment:
Not a big deal, but this could have been left out. (And introduced in a separate PR for BSD support later). Try to keep things as small and safe as possible. We don't have automated testing on BSD, so I'm not 100% sure this doesn't break the build for someone.